crates/trie/trie/src/witness.rs

```diff
@@ -2,10 +2,11 @@ use crate::{
     hashed_cursor::HashedCursorFactory, prefix_set::TriePrefixSetsMut, proof::Proof,
     trie_cursor::TrieCursorFactory, HashedPostState,
 };
+use alloy_primitives::{keccak256, Bytes, B256};
 use alloy_rlp::{BufMut, Decodable, Encodable};
 use itertools::Either;
 use reth_execution_errors::{StateProofError, TrieWitnessError};
-use reth_primitives::{constants::EMPTY_ROOT_HASH, keccak256, Bytes, B256};
+use reth_primitives::constants::EMPTY_ROOT_HASH;
```

Comment:
if / when we want to introduce an `alloy_consensus` dep we could use that here. Not necessary to do now though

Reply:
Yeah I was wondering, maybe we could do that in a second pass after cleaning up `alloy-primitives` stuffs.